setup.cfg

```diff
@@ -26,11 +27,10 @@ install_requires =
     pandas
     matplotlib
     scipy
-    ipython
     plotly
     scikit-learn
     antspyx
-    tensorflow <2.7
+    tensorflow
     jupyter
```

Comment:
given that jupyter is not needed for package itself but only for the demos, it might more sense to have it set up as an extra install.